source/extensions/filters/http/cache/cache_filter.md

```diff
@@ -0,0 +1,41 @@
+### HTTP Cache Filter
```

Comment:
Right now all of our dev docs are in /source/docs for better worse. WDYT about putting this in /source/docs/filters/http/cache/... or something like that? @jmarantz any thoughts on this? I'm fine either way. In general if we thing dev docs should live closer to the code they talk about maybe we should move the other docs as a separate thing. I could go either way on this.

Reply:
Done. (I think I misunderstood an earlier suggestion.)